File: BungiiAutomation/Bungii.Test.Regression.Android.Integration/Pages/Bungii/BungiiAcceptedPage.cs
using OpenQA.Selenium;
using OpenQA.Selenium.Support.PageObjects;

namespace Bungii.Test.Regression.Android.Integration.Pages.Bungii
{
    class BungiiAcceptedPage
    {
        public BungiiAcceptedPage(IWebDriver driver)
        {
            PageFactory.InitElements(driver, this);
        }

        [FindsBy(How = How.XPath, Using = "//android.widget.TextView[@text='BUNGII ACCEPTED']")]
        public IWebElement PageTitle_BungiiAccepted { get; set; }

        [FindsBy(How = How.XPath, Using = "//android.widget.LinearLayout[@id='com.bungii.customer:id/searching_view_bungii_accepted_container']/android.widget.TextView[@instance='1']")]
        public IWebElement Label_BungiiAccepted { get; set; }

        [FindsBy(How = How.Id, Using = "com.bungii.customer:id/bungii_accepted_enroute")]
        public IWebElement Label_DriverEnRoute { get; set; }

        [FindsBy(How = How.Id, Using = "com.bungii.customer:id/bungii_accepted_driver_image")]
        public IWebElement Image_Driver { get; set; }

        [FindsBy(How = How.Id, Using = "com.bungii.customer:id/rating_bar")]
        public IWebElement DriverRatingBar { get; set; }

        [FindsBy(How = How.Id, Using = "com.bungii.customer:id/bungii_accepted_driver_name")]
        public IWebElement Label_DriverName { get; set; }

        [FindsBy(How = How.Id, Using = "com.bungii.customer:id/bungii_accepted_button_ok")]
        public IWebElement Button_OK { get; set; }
    }
}

File: Bungii.Test.Integration/src/test/java/com/bungii/android/pages/bungii/BungiiAcceptedPage.java
package com.bungii.android.pages.bungii;

import com.bungii.common.core.PageBase;
import org.openqa.selenium.WebElement;

public class BungiiAcceptedPage extends PageBase {

    public WebElement PageTitle_BungiiAccepted()
    {
        return findElement("//android.widget.TextView[@text='BUNGII ACCEPTED']", LocatorType.XPath);
    }

    public WebElement Label_BungiiAccepted()
    {
        return findElement("//android.widget.LinearLayout[@id='com.bungii.customer:id/searching_view_bungii_accepted_container']/android.widget.TextView[@instance='1']", LocatorType.XPath);
    }
    public WebElement Label_DriverEnRoute()
    {
        return findElement("com.bungii.customer:id/bungii_accepted_enroute", LocatorType.Id);
    }

    public WebElement Image_Driver()
    {
        return findElement("com.bungii.customer:id/bungii_accepted_driver_image", LocatorType.Id);
    }
    public WebElement DriverRatingBar()
    {
        return findElement("com.bungii.customer:id/rating_bar", LocatorType.Id);
    }
    public WebElement Label_DriverName()
    {
        return findElement("com.bungii.customer:id/bungii_accepted_driver_name", LocatorType.Id);
    }
    public WebElement Button_OK()
    {
        return findElement("com.bungii.customer:id/bungii_accepted_button_ok", LocatorType.Id);
    }

}
